RELEASE-NOTES.md

```diff
@@ -46,6 +46,12 @@ The Axom project release numbers follow [Semantic Versioning](http://semver.org/
   and by calling `Shaper::setPercentError()` to set a refinement error percentage.
 
 ### Changed
+- The Axom library has been broken down into its component libraries (prefixed with `axom_`).
+  This change requires no change to downstream CMake users who import our targets.
+  The exported CMake target `axom` includes all components, but others will need to create the link line
```

Comment:
what does 'others' refer to here?

Reply:
people who don't import our targets. I'll clarify...